### PR TITLE
(DOCSP-12904): Link to Supplementary Examples - iOS SDK

### PR DESCRIPTION
**Pull request by mongomoe**
### Jira

- https://jira.mongodb.org/browse/DOCSP-12904

### Staged Changes (Requires MongoDB Corp SSO)

- [Additional Examples - iOS SDK](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-12904-Link-to-supplementary-examples/sdk/ios/additional-examples/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/ios/additional-examples.txt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+There are many examples of applications that the {+realm+} community has
```
Comment (nathan-contino-mongo) [inline source/sdk/ios/additional-examples.txt:17]:
```suggestion
There are many examples of applications that the {+service-short+} community has
```
The "realm" substitution is just for the database/data structure, not the product or community.

File: source/sdk/ios/additional-examples.txt [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+.. note:: 
+
+   The MongoDB Documentation team does not directly maintain these examples. 
+
+Examples by the Realm Engineering Team
+--------------------------------------
+You can find :github:`example applications
+<realm/realm-cocoa/tree/master/examples>` for both iOS and macOS that
+demonstrate how to use many Realm features, such as migrations, encryption, use
+of {+realm+} with UITableViewControllers, and more.
```
Comment (nathan-contino-mongo) [inline source/sdk/ios/additional-examples.txt:30]:
```suggestion
of {+realm+} with ``UITableViewController``, and more. 
```
Should this be monospaced?

File: source/sdk/ios/additional-examples.txt (hunk 1 above)
Comment (nathan-contino-mongo) [reply source/sdk/ios/additional-examples.txt:17]: The introductory sentence reads a bit awkwardly to me. I wonder if we could rephrase it to something like "Every day, the MongoDB community creates additional applications that demonstrate {+service-short+}." Or maybe that is too marketing-y?

**Review by nathan-contino-mongo — CHANGES_REQUESTED** [review]
Couple of nits, looking good though. I need to create one of these for Android!

File: source/sdk/ios/additional-examples.txt [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+.. note:: 
+
+   The MongoDB Documentation team does not directly maintain these examples. 
+
+Examples by the Realm Engineering Team
+--------------------------------------
+You can find :github:`example applications
+<realm/realm-cocoa/tree/master/examples>` for both iOS and macOS that
+demonstrate how to use many Realm features, such as migrations, encryption, use
+of {+realm+} with ``UITableViewController``, and more.
```
Comment (nathan-contino-mongo) [inline source/sdk/ios/additional-examples.txt:28]: Since "use of realm with UITableViewController" isn't a "Realm feature" as you introduce this list, maybe you could rewrite this into two sentences? Also, be sure to use the {+service-short+} source constant for all mentions of Realm in paragraph text.

```
The :github:`realm-cocoa example applications
<realm/realm-cocoa/tree/master/examples>` provide examples for both iOS and macOS that
demonstrate how to use {+service-short+} features such as migrations, encryption,
and {+sync+}. The examples also demonstrate how to combine {+service-short+} with
common iOS concepts like ``UITableViewController``. 
```

File: source/sdk/ios/additional-examples.txt [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+The Realm community has created many applications that demonstrate the usage of the Realm iOS SDK. This page contains some of these applications, and the MongoDB documentation team will update this page with additional examples over time.
```
Comment (nathan-contino-mongo) [inline source/sdk/ios/additional-examples.txt:17]: Always use the source constants for the product name.
```suggestion
The {+service-short+} community has created many applications that demonstrate the usage of the Realm iOS SDK. This page contains some of these applications, and the MongoDB documentation team will update this page with additional examples over time.
```

Comment (nathan-contino-mongo) [inline source/sdk/ios/additional-examples.txt:17]: In the note below, you mention the "MongoDB Documentation team", but in this sentence, you write it as "MongoDB documentation team" (no capital 'D' on "documentation"). We should keep this identifier consistent.

Comment (nathan-contino-mongo) [inline source/sdk/ios/additional-examples.txt:17]: Can we cut off this line at 80 characters? It's super long.

**Review by nathan-contino-mongo — CHANGES_REQUESTED** [review]
Some source constant and consistency issues still.

File: source/sdk/ios/additional-examples.txt (hunk 4 above)
Comment (mongomoe) [reply source/sdk/ios/additional-examples.txt]: Fixed

Comment (mongomoe) [reply source/sdk/ios/additional-examples.txt]: fixed 👍

File: source/sdk/ios/additional-examples.txt (hunk 3 above)
Comment (mongomoe) [reply source/sdk/ios/additional-examples.txt]: Yup that sounds good, copied almost verbatim except I changed "... features such as migrations, encryption,
and {+sync+}" to remove Realm Sync, since the iOS examples in that project do not contain Realm Sync examples.

File: source/sdk/ios/additional-examples.txt [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+.. note:: 
+
+   The MongoDB Documentation team does not directly maintain these examples. 
+
+Examples by the Realm Engineering Team
+--------------------------------------
+The :github:`realm-cocoa example applications
+<realm/realm-cocoa/tree/master/examples>` provide examples for both iOS and
+macOS that demonstrate how to use {+service-short+} features such as migrations,
```
Comment (nathan-contino-mongo) [inline source/sdk/ios/additional-examples.txt:30]: unneeded comma
```suggestion
macOS that demonstrate how to use {+service-short+} features such as migrations
```

**Review by nathan-contino-mongo — APPROVED** [review]
LGTM with one nit
